doc/stdlib/sort.rst

```diff
@@ -116,60 +116,58 @@ specified via the ``lt`` keyword.
 Sorting Functions
 -----------------
 
-.. function:: sort!(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+.. function:: sort!(v; alg::Algorithm=defalg(v), lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
 
    .. Docstring generated from Julia source
 
    Sort the vector ``v`` in place. ``QuickSort`` is used by default for numeric arrays while ``MergeSort`` is used for other arrays. You can specify an algorithm to use via the ``alg`` keyword (see Sorting Algorithms for available algorithms). The ``by`` keyword lets you provide a function that will be applied to each element before comparison; the ``lt`` keyword allows providing a custom "less than" function; use ``rev=true`` to reverse the sorting order. These options are independent and can be used together in all possible combinations: if both ``by`` and ``lt`` are specified, the ``lt`` function is applied to the result of the ``by`` function; ``rev=true`` reverses whatever ordering specified via the ``by`` and ``lt`` keywords.
 
-.. function:: sort(v, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+.. function:: sort(v; alg::Algorithm=defalg(v), lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
 
    .. Docstring generated from Julia source
 
-   Variant of ``sort!`` that returns a sorted copy of ``v`` leaving ``v`` itself unmodified.
+   Variant of :func:`sort!` that returns a sorted copy of ``v`` leaving ``v`` itself unmodified.
 
-.. function:: sort(A, dim, [alg=<algorithm>,] [by=<transform>,] [lt=<comparison>,] [rev=false])
+.. function:: sort(A, dim::Integer; alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward, initialized::Bool=false)
 
    .. Docstring generated from Julia source
 
-   Sort a multidimensional array ``A`` along the given dimension.
+   Sort a multidimensional array ``A`` along the given dimension. ``lt`` defines the comparison to use.
```

Comment:
Should probably cross-reference `sort!` for the keyword arguments that are shared? and describe the ones that aren't, I think just `initialized`.